init.ps1

```diff
@@ -25,4 +25,11 @@ Write-Host "Restoring Monaco Editor"
 ExecSafe { & $PSScriptRoot\tools\Restore-MonacoEditor.ps1 -RootFolder $PSScriptRoot }
 
 Write-Host "Done."
-Write-Output "---------------------------------------"
\ No newline at end of file
+Write-Output "---------------------------------------"
+
+# Install nodes modules
+Write-Host "Installing node modules"
+Get-ChildItem $PSScriptRoot\src\ -rec |? { $_.FullName.EndsWith('DevToys.Blazor.csproj') } |% {
+    Write-Host "Installing node modules for $($_)..."
+    ExecSafe { & npm install $_.Directory  }
```

Comment:
Why doing this? We already do this on `dotnet restore`. See here:
https://github.com/DevToys-app/DevToys/blob/79c33ce3f6bfe90f737e02d7b5b2adfe9accbc22/src/app/dev/DevToys.Blazor/DevToys.Blazor.csproj#L108-L119